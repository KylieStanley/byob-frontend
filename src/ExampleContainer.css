pre {
  text-align: left;
  white-space: pre-wrap;
  margin: 40px;
}