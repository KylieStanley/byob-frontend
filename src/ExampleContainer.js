import React from 'react'

const ExampleContainer = () => {
  return (
    <h2></h2>
  )
}

export default ExampleContainer