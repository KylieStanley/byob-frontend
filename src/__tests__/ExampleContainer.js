import React from 'react'
import { shallow } from 'enzyme'
import  ExampleContainer from '../ExampleContainer'


describe('ExampleContainer', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallow(<ExampleContainer example={{}} />)
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
