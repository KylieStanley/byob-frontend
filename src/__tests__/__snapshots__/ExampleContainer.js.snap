// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExampleContainer should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ExampleContainer
    example={Object {}}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h2>
        Select an example response above
      </h2>,
      "className": "example-container",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Select an example response above",
      },
      "ref": null,
      "rendered": "Select an example response above",
      "type": "h2",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h2>
          Select an example response above
        </h2>,
        "className": "example-container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Select an example response above",
        },
        "ref": null,
        "rendered": "Select an example response above",
        "type": "h2",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
